cli/cmd/cluster_create.go

```diff
@@ -35,8 +35,8 @@ https://docs.replicated.com/vendor/testing-how-to#limitations`,
 	cmd.Flags().StringVar(&r.args.createClusterKubernetesDistribution, "distribution", "", "Kubernetes distribution of the cluster to provision")
 	cmd.Flags().StringVar(&r.args.createClusterKubernetesVersion, "version", "", "Kubernetes version to provision (format is distribution dependent)")
 	cmd.Flags().IntVar(&r.args.createClusterNodeCount, "nodes", int(1), "Node count")
-	cmd.Flags().IntVar(&r.args.createClusterMinNodeCount, "min-nodes", int(-1), "Minimum Node count (only for EKS, AKS and GKE clusters)")
-	cmd.Flags().IntVar(&r.args.createClusterMaxNodeCount, "max-nodes", int(-1), "Maximum Node count (only for EKS, AKS and GKE clusters)")
+	cmd.Flags().IntVar(&r.args.createClusterMinNodeCount, "min-nodes", int(-1), "Minimum Node count (only for EKS, AKS and GKE clusters). A negative value will be ignored.")
```

Comment:
I particularly like how docker cli handles this in the run command

```
      --memory-swappiness int          Tune container memory swappiness (0 to 100) (default -1)
```

https://github.com/docker/cli/blob/a1361a1372adc0dff6016927a047bff71717a8d4/cli/command/container/opts.go#L304